internal/constants.go

```diff
@@ -13,4 +13,5 @@ const (
 
 const (
 	SizeOfBlock = 512
+	ClusterSize = 4096
```

Comment:
Let's rename these constants as `UNIXSizeOfBlock` and `NSClusterSize` to prevent confusion